build/source/dshare/globalData.f90

```diff
@@ -55,6 +55,10 @@ MODULE globalData
  ! * part 1: parameters that are fixed across multiple instantiations
  ! ----------------------------------------------------------------------------------------------------------------
 
+ ! define version number
+ character(len=256),parameter,public         :: summaVersion='v2.34.56'
+`
```

Comment:
Should the summa version also be included in control files? This implementation is not correct.